bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -27,11 +27,11 @@
 			<channel id="upb_link_deactivated_event" typeId="upb_link_deactivated_event"/>
 		</channels>
 		<properties>
-			<property name="model number"/>
-			<property name="major version"/>
-			<property name="minor version"/>
+			<property name="modelNumber"/>
+			<property name="majorVersion"/>
+			<property name="minorVersion"/>
 			<property name="revision"/>
-			<property name="phone number"/>
+			<property name="phoneNumber"/>
```

Comment:
For dynamically set properties you don't need to define them here. This will result in them being shown as empty properties until they will be set by code.

Reply:
Are they set in code? Since renamed, question is if code should be adapted as well, or you renamed them here to align with names in current code? Besides that, if these are firmware properties, modelNumber should be renamed to **modelId** and majorVersion/minorVersion/revision could be combined in **firmwareVersion**.

Reply:
They are set in code, and it was just to align them with the code. I like your idea of using the builtin openhanded properties though so I switch to that.

Reply:
Fixed, removed the properties from XML files as they are all dynamically set in code.